package.json

```diff
@@ -1,84 +1,90 @@
 {
-    "name": "html-flip-book",
-    "version": "1.0.0",
-    "description": "Flipbook component for HTML",
-    "homepage": "https://github.com/doradsoft/html-flip-book",
-    "repository": {
-      "type": "git",
-      "url": "git+https://github.com/doradsoft/html-flip-book.git"
+  "name": "html-flip-book",
+  "version": "1.0.0",
+  "description": "Flipbook component for HTML",
+  "homepage": "https://github.com/doradsoft/html-flip-book",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/doradsoft/html-flip-book.git"
+  },
+  "bugs": {
+    "url": "https://github.com/doradsoft/html-flip-book/issues"
+  },
+  "engines": {
+    "node": ">=20"
+  },
+  "scripts": {
+    "bundle": "npm run format:write && npm run package",
+    "ci-test": "npx jest",
+    "coverage": "npx make-coverage-badge --output-path ./badges/coverage.svg",
+    "format:write": "npx prettier --write .",
+    "format:check": "npx prettier --check .",
+    "lint": "npx eslint . -c ./eslint.config.mjs",
+    "package": "npx ncc build src/index.ts -o dist --source-map --license licenses.txt",
+    "package:watch": "npm run package -- --watch",
+    "test": "npx jest",
+    "all": "npm run format:write && npm run lint && npm run test && npm run coverage && npm run package"
+  },
+  "compilerOptions": {
```

Comment:
1. `package.json` is not the place for configuring typescript compiler options. 
2. Configuration is wrong is we use ES modules.
So just delete this...